code/modules/mob/new_player/sprite_accessories.dm

```diff
@@ -711,6 +711,16 @@
 	icon_state = "female_blackalt"
 	gender = FEMALE
 
+/datum/sprite_accessory/underwear/female_white_neko
+	name = "Ladies White Neko"
+	icon_state = "female_neko_white"
+	gender = FEMALE
+
+/datum/sprite_accessory/underwear/female_black_neko
+	name = "Ladies Black Neko"
+	icon_state = "female_neko_black"
+	gender = FEMALE

```

Comment:
upon further review, it seems like most female underpants don't take dixel considerations in, so just merge it as is, and I'll fix it later for all of them.